.drone.star

```diff
@@ -397,7 +397,11 @@ def binary(ctx, name):
           'files': [
             'dist/release/*',
           ],
-          'title': ctx.build.ref.replace("refs/tags/", ""),
+          'checksum': [
+            'md5',
+            'sha256'
+          ],
+          'title': ctx.build.ref.replace("refs/tags/v", ""),
```

Comment:
This should also be reverted, IMHO it's totally fine to keep the `v` prefix for the release title.

Reply:
If we compare with the Releases of other Repos, we don't include the `v` there.
We are using the `v` prefix only for the tag itself. Referring to Semver, a release Version has no prefix, but the tag has. What do you think?

In the end maybe a matter of taste?

Reply:
Than you got to see what others say. I personally like the v prefix :)

Reply:
There are strong opinios against `v` in Release titles 🤷‍♂